descriptions/rep/en_us.lua

```diff
@@ -522,7 +522,7 @@ local repTrinkets={
 	[149] = {"149", "Panic Button", "Activates your active item upon taking damage"}, -- Panic Button
 	[150] = {"150", "Blue Key", "Entering a room that requires a key brings you in a room resembling the Hush stage#Room acts as an in-between of the two rooms"}, -- Blue Key
 	[151] = {"151", "Flat File", "Causes spikes to retract, making them harmless#Also affects {{CursedRoom}}Curse Room doors, mimics and any spike obstacle"}, -- Flat File
-	[152] = {"152", "Telescope Lens", "Increases the default chance for a {{Planetarium}}Planetarium to spawn to 15%#Adds another 10% chance#More than one Planetarium can spawn#Planetariums can now spawn in the Womb and Corpse"}, -- Telescope Lens	
+	[152] = {"152", "Telescope Lens", "+24% {{Planetarium}}Planetarium chance if none have spawned#+9% if a Planetarium has already spawned#Planetariums can now spawn in the Womb and Corpse"}, -- Telescope Lens	
```

Comment:
The previous description was correct, no?

https://bindingofisaacrebirth.fandom.com/wiki/Telescope_Lens

https://github.com/wofsauge/External-Item-Descriptions/pull/180#issuecomment-966525650

Reply:
The percentages were swapped (I just checked in game, it goes to 10% after you've entered one), and it wasn't clear what the "adds another x% chance" was for (sounded like both chances are always active, rather than one applying only to the first planetarium), and it's already possible for "More than one Planetarium can spawn" (albeit at 1%) so that's not really an item attribute